Source/WebCore/platform/ios/VideoPresentationInterfaceAVKit.mm

```diff
@@ -845,13 +857,20 @@ - (WTFLogChannel*)logChannel
     if (!m_routingContextUID.isEmpty())
         [m_playerViewController setWebKitOverrideRouteSharingPolicy:(NSUInteger)m_routeSharingPolicy routingContextUID:m_routingContextUID];
 
+    if (!m_currentMode.hasPictureInPicture() && !m_changingStandbyOnly) {
+        ALWAYS_LOG_IF_POSSIBLE(LOGIDENTIFIER, "Moving videoView to fullscreen WebAVPlayerLayerView");
+        [playerLayerView() transferVideoViewTo:[m_playerViewController playerLayerView]];
+    }
+
 #if PLATFORM(WATCHOS)
     m_viewController = videoPresentationModel() ? videoPresentationModel()->createVideoFullscreenViewController(avPlayerViewController()) : nil;
 #endif
 }
 
 void VideoPresentationInterfaceAVKit::invalidatePlayerViewController()
 {
+    returnVideoView();
```

Comment:
I know this isn't new, but I had to go to the implementation to know what this does. Maybe `returnVideoViewToPlayerLayerView`?